rotkehlchen/api/v1/schemas.py

```diff
@@ -2318,3 +2314,14 @@ def validate_schema(  # pylint: disable=no-self-use
                 message='Can\'t delete the etherscan node',
                 field_name='identifier',
             )
+
+
+class OptionalAddressesListSchema(AsyncQueryArgumentSchema):
+    addresses = fields.List(EthereumAddressField(required=True), load_default=None)
+
+
+class DetectTokensSchema(
+    OptionalAddressesListSchema,
+    OnlyCacheQuerySchema,
```

Comment:
As I explained above.

```suggestion
    OnlyCacheQuerySchema,
    AsyncQueryArgumentSchema,
```